package.json

```diff
@@ -93,10 +95,13 @@
     "comment-json": "1.1.3",
     "coveralls": "3.0.1",
     "doctoc": "1.3.1",
-    "eslint-config-amo": "1.7.0",
+    "eslint-config-amo": "1.8.3",
+    "eslint-plugin-amo": "^1.7.0",
     "eslint-plugin-async-await": "0.0.0",
     "eslint-plugin-jest": "21.22.0",
     "eslint-plugin-promise": "4.0.0",
+    "eslint-config-prettier": "3.0.1",
```

Comment:
@EnTeQuAk it may be better if we keep these deps in the order that npm install expects (which is in alphabetical order, if I'm not wrong), otherwise the next `npm install --save-dev ...` is going to reorder the ones that are not in the order that npm expects (e.g. in a greenkeeper PR).

Reply:
Done in ff5d6fe